### PR TITLE
Cmake stats and editor

### PR DESCRIPTION
**Pull request by therearesomewhocallmetim**

### CODE REVIEW

File: stats/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,55 @@
+include_directories(${OMIM_DIR} ${OMIM_DIR}/3party/Alohalytics/src)
+
+set(
+  SRC
+  ${OMIM_DIR}/3party/Alohalytics/src/cpp/alohalytics.cc
```
Comment (syershov) [inline stats/CMakeLists.txt]: s/OMIM_DIR/OMIM_ROOT/

File: stats/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+else()
+  set(
+    SRC
+    ${SRC}
+    ${OMIM_DIR}/3party/Alohalytics/src/posix/file_manager_posix_impl.cc
+  )
+endif()
+
+
+if (${PL_IPHONE} OR ${PL_MAC})
```
Comment (syershov) [inline stats/CMakeLists.txt]: s/PL_/PLATFORM_/g

File: 3party/liboauthcpp/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+add_compile_options(
+  "-Wall"
+)
+
+
+include_directories(include src)
+
+set(
+  SRC
+    include/liboauthcpp/liboauthcpp.h
```
Comment (ygorshenin) [inline 3party/liboauthcpp/CMakeLists.txt]: Wrong indent.

File: editor/editor_tests/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  coding
+  base
+  stats_client
+  opening_hours
+  pugixml
+  oauthcpp
+  tomcrypt
+)
+
+  target_link_libraries(
```
Comment (ygorshenin) [inline editor/editor_tests/CMakeLists.txt]: Wrong indent.

File: editor/editor_tests/CMakeLists.txt [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  osm_feature_matcher_test.cpp
+  ui2oh_test.cpp
+  user_stats_test.cpp
+  xml_feature_test.cpp
+)
+
+omim_add_test(editor_tests ${SRC})
+
+omim_link_libraries(
+  editor_tests
```
Comment (ygorshenin) [inline editor/editor_tests/CMakeLists.txt]: This code should be executed iff editor_test target is defined.

Comment (therearesomewhocallmetim) [inline editor/editor_tests/CMakeLists.txt]: See `function(omim_link_libraries target)` in root cmake file.

Comment (ygorshenin) [inline editor/editor_tests/CMakeLists.txt]: Okay, seems good.

File: 3party/liboauthcpp/CMakeLists.txt [hunk 6]
```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(oauthcpp C CXX)
+
+add_compile_options(
+  "-Wall"
+)
+
+
```
Comment (ygorshenin) [inline 3party/liboauthcpp/CMakeLists.txt]: Remove this blank line, please.

File: stats/CMakeLists.txt [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  )
+else()
+  set(
+    SRC
+    ${SRC}
+    ${OMIM_ROOT}/3party/Alohalytics/src/posix/file_manager_posix_impl.cc
+  )
+endif()
+
+
```
Comment (ygorshenin) [inline stats/CMakeLists.txt]: Remove this blank line, please.

**Comment by ygorshenin** [comment]
LGTM

**Comment by therearesomewhocallmetim** [comment]
PTAL

File: 3party/liboauthcpp/CMakeLists.txt [hunk 8]
```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(oauthcpp C CXX)
+
+add_compile_options(
+  "-Wall"
```
Comment (therearesomewhocallmetim) [inline 3party/liboauthcpp/CMakeLists.txt]: Done

File: stats/CMakeLists.txt (hunk 2 above)
Comment (therearesomewhocallmetim) [inline stats/CMakeLists.txt]: Done

File: 3party/liboauthcpp/CMakeLists.txt (hunk 3 above)
Comment (therearesomewhocallmetim) [inline 3party/liboauthcpp/CMakeLists.txt]: Done

File: 3party/liboauthcpp/CMakeLists.txt (hunk 6 above)
Comment (therearesomewhocallmetim) [inline 3party/liboauthcpp/CMakeLists.txt]: Done

File: stats/CMakeLists.txt (hunk 7 above)
Comment (therearesomewhocallmetim) [inline stats/CMakeLists.txt]: Done

**Pull request merged**